### DISCUSSION TITLE
Pullrequest from github desktop when signed up shouldn't display 404

### DISCUSSION THREAD

**Discussion by tuktukjohansson** [community/community#73357] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Pullrequest from Github Desktop shouldn't bring you to a page displaying 404 if you're not logged in right? 404 stands for error and isn't a pleasant welcoming. I've just been logged out, just say something like "Loggin session expired, please log in again to continue" :)

**Comment by esurov** [comment]
It was just a GH outage yesterday.
